networking/p2p/src/main/java/tech/pegasys/artemis/networking/p2p/HobbitsSubProtocolHandler.java

```diff
@@ -79,11 +103,7 @@ public AsyncCompletion stop() {
 
   @Subscribe
   public void onNewUnprocessedBlock(BeaconBlock block) {
```

Comment:
Add this log message:
`LOG.log(Level.INFO, "Gossiping new block with state root: " + block.getState_root());`